pypeit/spectrographs/keck_mosfire.py

```diff
@@ -347,7 +372,214 @@ def configuration_keys(self):
             and used to constuct the :class:`~pypeit.metadata.PypeItMetaData`
             object.
         """
-        return ['decker', 'dispname', 'filter1']
+        return ['decker_basename', 'slitlength', 'slitwid', 'dispname', 'filter1']
+
+    def modify_config(self, fitstbl, cfg):
```

Comment:
Provide some context for why this method is necessary in the docs. I presume this is something to do with reducing long2pos data with calibrations taken with a different decker. If that is the case explicitly state please.

Reply:
Added.